### PR TITLE
Add more Wandering Trader API

### PR DESCRIPTION
**Pull request by HexedHero**
Allows you to set if the Wandering Trader should drink his invisibility potion when its night or his milk when its day

First time doing API, tested with a plugin to spawn two entities one with normal behaviour and one with these two settings set to false and it works as intended.

### CODE REVIEW

**Comment by kennytv** [comment]
I would remove the `get` prefix from the two `getCan...` methods, not sure if that's just me tho.

File: Spigot-Server-Patches/0648-Add-Wandering-Trader-API.patch [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     public EntityType getType() {
+         return EntityType.WANDERING_TRADER;
+     }
++
++    // Paper start - Add WanderingTrader API
++    @Override
++    public void setCanDrinkPotion(boolean bool) {
++        getHandle().canDrinkPotion = bool;
++    }
++
```
Comment (MiniDigger) [inline Spigot-Server-Patches/0648-Add-Wandering-Trader-API.patch]:
```suggestion
+
+    @Override
```

File: Spigot-Server-Patches/0648-Add-Wandering-Trader-API.patch [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
++    public boolean getCanDrinkPotion() {
++        return getHandle().canDrinkPotion;
++    }
++
```
Comment (MiniDigger) [inline Spigot-Server-Patches/0648-Add-Wandering-Trader-API.patch]:
```suggestion
+
+    @Override
```

**Review by MiniDigger — APPROVED** [review]

**Comment by HexedHero** [comment]
Rebased and done Kenny's and Mini's requested changes

**Review by kennytv — APPROVED** [review]
Thank you

**Pull request merged**